reference/glossary/index.md

```diff
@@ -264,6 +264,26 @@ $$$glossary-elastic-security-indices$$$ {{elastic-sec}} indices
 $$$glossary-elastic-stack$$$ {{stack}}
 :   Also known as the *ELK Stack*, the {{stack}} is the combination of various Elastic products that integrate for a scalable and flexible way to manage your data.
 
+$$$glossary-elasticsearch$$$ {{es}}
+:   The open source search and analytics engine, data store, and vector database which is the core of the [{{stack}}](#glossary-elastic-stack) and powers the Elastic platform.
+:::{admonition} Disambiguation
```

Comment:
nitpick: disambiguation is a big word. how does "What else can Elasticsearch mean?" or something a little friendlier strike you?